packages/components/docs/token-table.tsx

```diff
@@ -1,9 +1,6 @@
-import classNames from "classnames/bind";
```

Comment:
`classnames` was being brought in through a storybook package, but with these updates it's no longer in the dependency tree.

The `no-extraneous-dependencies` should've stopped us from bringing this in without having an explicit dependency on it. Maybe that rule isn't turned on by default with the imports plugin? 🤔

Reply:
Added the rule in #697